Trimble.Modus.Core/BaseComponent.cs

```diff
@@ -1,8 +1,12 @@
-﻿namespace Trimble.Modus.Core
```

Comment:
Please remove the .Core project if it is not necessary

Reply:
Removed .Core Project